services/src/contexts/postgres.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(feature = "postgres")]
```

Comment:
You exclude it once in `mod.rs` and once here, wouldn't be one in/exclusion enough?

Reply:
I guess it would be enough to exclude the whole file here. It wasn't enough to just do it in `mod.rs`. I guess it makes sense to exclude it in both places. If I only exclude it here, the file will (probably) still be accessed during compilation.

Reply:
i guess you have to add cfg(feature = "postgres") in mod.rs before mod postgres.

Reply:
26511745f68be89453766afe268f7d49343fa48b